tests/test_models_circuit_execution.py

```diff
@@ -7,7 +7,7 @@
 def test_eager_execute(backend, accelerators):
     c = Circuit(4, accelerators)
     c.add(gates.H(i) for i in range(4))
-    final_state = backend.execute_circuit(c)
+    final_state = backend.execute_circuit(c)._state
```

Comment:
As in previous comments, prefer
```suggestion
    final_state = backend.execute_circuit(c).state()
```
outside the class.